model/src/main/java/userinfo/ClientInfo.java

```diff
@@ -0,0 +1,44 @@
+package userinfo;
```

Comment:
I think we have put all the UserInfo classes and subclasses in the user package as well. You might want to move these files before we merge

Reply:
i actually didnt do anything with these classes, so im not sure why they are included in my branch/PR. my best guess is that i had set up my branch incorrectly but i have no idea.

should i still make changes to this file?

Reply:
Yeah it does seem like the branch might not be set up correctly

Reply:
For me it shows `package com.javahelp.model.user;`